server/engine/src/engine.py

```diff
@@ -52,43 +57,56 @@ def main():
         # segments that have not yet thrown an error during loading.
         print('Running kinematics fit', flush=True)
         subject.run_kinematics_fit(DATA_FOLDER_PATH)
+        # This will create a B3D object in memory for the current fit of the subject. This can be used at any point to
+        # write out the B3D file, but also can be used as our working object as we run subsequent pipeline steps.
+        subject_on_disk: nimble.biomechanics.SubjectOnDisk = subject.create_subject_on_disk(href)
 
-        # This plausibly skips kinematics, with placeholder 0s for (most of) the necessary kinematics results.
-        # for trial in subject.trials:
-        #     for segment in trial.segments:
-        #         segment.kinematics_status = ProcessingStatus.FINISHED
-        #         num_steps = len(segment.marker_observations)
-        #         segment.kinematics_poses = np.zeros((subject.skeleton.getNumDofs(), num_steps))
-        #         dummy_results: nimble.biomechanics.MarkerInitialization = nimble.biomechanics.MarkerInitialization()
-        #         dummy_results.poses = segment.kinematics_poses
-        #         dummy_results.updatedMarkerMap = subject.markerSet
-        #         dummy_results.groupScales = subject.skeleton.getGroupScales()
-        #         dummy_results.joints = []
-        #         dummy_results.jointWeights = np.zeros(0)
-        #         dummy_results.axisWeights = np.zeros(0)
-        #         dummy_results.jointAxis = np.zeros((0, num_steps))
-        #         dummy_results.jointCenters = np.zeros((0, num_steps))
-        #         segment.marker_fitter_result = dummy_results
-
-        # This will lowpass filter all the input kinematics data
-        print('Running lowpass filter', flush=True)
-        subject.lowpass_filter()
-
-        # The dynamics fit will fit the dynamics parameters of the subject, to all the trial segments that have not yet
-        # thrown an error during loading or kinematics fitting.
-        if not subject.disableDynamics:
-            print('Running dynamics fit', flush=True)
-            subject.run_dynamics_fit()
-        # This will write out a folder of OpenSim results files.
+        print('Running acc minimizing pass...', flush=True)
+        add_acc_minimize_pass(subject_on_disk)
```

Comment:
For clarity, I'm partial using full words in these methods (i.e., `add_acceleration_minimize_pass`). Bonus points for using a more active terms for these methods (e.g., `run_acceleration_minimizer/smoother`, `detect_missing_grfs`, `run_dynamics_pass`, etc.).

Reply:
Also, consider renaming the test files accordingly.

Reply:
Done